addon/adapters/osf-adapter.js

```diff
@@ -80,7 +85,7 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
         if (Object.keys(snapshot.record.changedAttributes()).length) {
             promises.push(this._super(...arguments));
         }
-        return Ember.RSVP.Promise.all(promises).then(values => {
+        return Ember.RSVP.Promise.allSettled(promises).then(values => {
```

Comment:
Not every value returned from this will be successful...

If I'm reading this correctly, then to use this method and update the store correctly we'll need to do something (like filtering?) to check based on the `state` parameter of the return object. See linked docs for info.